application/app/src/main/java/uk/nhs/digital/uec/api/authentication/model/AuthToken.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.uec.api.authentication.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public class AuthToken {
+
+  private String accessToken;
+  private String refreshToken;
+  private String message;
```

Comment:
What is this message field for?

Reply:
In case of Exception, I will just capture the exception message and populate the same object and the json will have the relevant field as response. In our case the message will be '401 - Unauthorised'.

Reply:
We don't need it, we can just return a 401 from the controlller.